docs/reference/rest_api_drafts_records.md

```diff
@@ -19,6 +19,7 @@ Used for interacting with unpublished or edited draft records.
 | `access`   | object | body     | [Access options](metadata.md#access) for the record. |
 | `files`    | object | body     | Files options (see below) for the record. |
 | `metadata` | object | body     | [Metadata](metadata.md#metadata) of the record. |
+| `custom_fields` | object | body     | [Custom fields](https://inveniordm.docs.cern.ch/customize/metadata/custom_fields/records/#declaring-custom-fields)) metadata for the record. (v10 and newer) |
```

Comment:
Let's use the internal linking and we should be good for this one and the other:

```suggestion
| `custom_fields` | object | body     | [Custom fields](../customize/metadata/custom_fields/records.md#declaring-custom-fields) metadata for the record. (v10 and newer) |
```

Reply:
Thanks, I've switched both

Reply:
Oh there was also a parenthesis fix in that suggestion (we need to remove that extra sneaky parenthesis) :ninja:

Reply:
Thanks, got them!